core/new-gui/src/app/workspace/component/operator-panel/operator-panel.component.ts

```diff
@@ -61,6 +79,55 @@ export class OperatorPanelComponent implements OnInit {
     this.operatorSchemaList = operatorMetadata.operators;
     this.groupNamesOrdered = getGroupNamesSorted(operatorMetadata.groups);
     this.operatorGroupMap = getOperatorGroupMap(operatorMetadata);
+
+
+
+    this.filteredOptions = this.myControl.valueChanges
+      .map(value => this._filter(value));
+    }
+
+  private _filter(value: string): string[] {
+
```

Comment:
change function name to something like `fuzzyFilter`. In addition, add some comments describing what this method is doing.